libs/blocks/merch-card/merch-card.js

```diff
@@ -357,8 +357,25 @@ const simplifyHrs = (el) => {
   hrs.forEach((hr) => {
     if (hr.parentElement.tagName === 'P') {
       hr.parentElement.replaceWith(hr);
+      hr.style.width = '100%';
```

Comment:
What does this do exactly, or why is it needed?

Reply:
The horizontal line will stretch across the entire width of the container it is inside.

Reply:
given
![image](https://github.com/user-attachments/assets/bd66ab58-28f1-45f3-9b7a-9d649018cccd)
shouldn't it as follows in merch-card.css ?

```css
merch-card .body-lower > hr {
width: 100%
}
```

Reply:
modifying styles eagerly has performance implication as it can cause a repaints.